qacode/core/enums/enum_base.py

```diff
@@ -13,9 +13,9 @@ class EnumBase(Enum):
     @classmethod
     def get_properties(cls):
         """Return enum values"""
-        return [item.value for item in type(cls)]
+        return [name for name, member in cls.__members__.items()]
 
     @classmethod
     def has_property(cls, value):
         """Returns True if enum have value"""
-        return any(value == item.value for item in cls)
+        return cls(value)

```

Comment:
Es un milagro que esto funcione...

```py
>>> from enum import Enum
>>> class MyEnum(Enum):
...     A = 1
...     B = 2
...
>>> MyEnum(1)
<MyEnum.A: 1>
>>> MyEnum(3)
ValueError: 3 is not a valid MyEnum

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.7/enum.py", line 309, in __call__
    return cls.__new__(cls, value)
  File "/usr/lib/python3.7/enum.py", line 561, in __new__
    raise exc
  File "/usr/lib/python3.7/enum.py", line 545, in __new__
    result = cls._missing_(value)
  File "/usr/lib/python3.7/enum.py", line 574, in _missing_
    raise ValueError("%r is not a valid %s" % (value, cls.__name__))
ValueError: 3 is not a valid MyEnum
```

Entiendo que lo correcto sería:
```suggestion
        try:
            cls(value)
        except ValueError:
            return False
        else:
            return True
```

Reply:
right, bad change and bad TEST to ensure change, doing again...